monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
 
-For Monero, RPC is already being used in a variety of ways. For example, the standard Monero daemon, containing most of Monero's functionality, is _monerod_. It implements the server side of RPC for a variety of blockchain-related function calls. And the program _monero-wallet-rpc_, delivered with the standard Monero installation, implements the server side of wallet functionality while itself being a client to a _monerod_ server. Monero RPC calls work much like a browser communicating with a web server. Data is sent using the POST method, and returned information is textual, usually JSON. If you are interested in detailed examples of RPC messages, please see [Example #1](https://www.monerooutreach.org/stories/RPC-Pay.html#box1) on using Monero RPC calls.
+Para Monero, el RPC ya está siendo utilizado de muchas maneras. Por ejemplo, el _monerod_, es el daemon estándard que contiene la mayoría de funcionalidades de Monero. Implementa el lado del servidor del RPC para una variedad de funciones relacionadas a las llamadas a la cadena de bloques. Y el programa _monero-wallet-rpc_, que se entrega con la instalación estándard de Monero, implementa el lado del servidor de las funciones del monedero, mientras que al mismo tiempo es un cliente a un servidor _monerod_. Las llamadas RPC de Monero trabajan muy parecido a la comunicación entre un navegador (browser) y un servidor web. Las información es envíada usando el método POST y devuelta como texto, usualmente JSON. Si estás interesado en ejemplos detallados en el uso de las llamadas RPC de Monero, mira el [Ejemplo #1](https://www.monerooutreach.org/stories/RPC-Pay.html#box1). 
 
-### _Mining Hashes_
+### _La minería de hashes_
 
-The mining process for Monero's RandomX Proof of Work (PoW) starts by executing a pseuorandom program that is a function of the new transactions, a nonce, a payment address, and data from the previous block. Executing a random program like this is easy for CPUs, but hard for ASICs. The output of the program is then cryptographically hashed into a 256-bit number. If this number is less than a specified threshold, then it counts as mining a new block.
+El proceso para Monero de minería prueba de trabajo (PoW) RandomX, arranca al ejecutarse un programa pseudoaleatorio (función de la nueva transacción), un nonce (número criptográfico arbitrario de un solo uso), una dirección de pago y datos del bloque anterior. Ejecutar un programa aleatorio como este es fácil para los ordenadores CPU, pero difícil para los ASIC. Luego, a la salida del programa, se le aplica una función hash criptográfica y convertida a un número de 256 bits. Si este número es menor en un umbral específico, cuenta como el haber minado un nuevo bloque.
```

Comment:
Another rephrasing suggestion:

_El proceso de minería para **el algoritmo RandomX Prueba de Trabajo (PoW) de Monero comienza al ejecutar** un programa pseudoaleatorio..._

Added _**el algoritmo**_ as a complement to make it more phrase-like. I think it can also be omitted.

Reply:
Added, fixed, and changed.